app/Console/Commands/CreateAccount.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Account\Account;
+use Illuminate\Console\Command;
+
+class CreateAccount extends Command
```

Comment:
Please add the `ConfirmableTrait` trait and a test like `$this->confirmToProceed()` (see [MigrateDatabaseCollation.php](https://github.com/monicahq/monica/blob/master/app/Console/Commands/MigrateDatabaseCollation.php) for instance)